app/models/ems_refresh/save_inventory.rb

```diff
@@ -165,7 +165,7 @@ def save_vms_inventory(ems, hashes, target = nil, disconnect = true)
         parent = vms[h.fetch_path(:parent_vm, :id)]
         child = vms[h[:id]]
 
-        child.with_relationship_type('genealogy') { child.parent = parent } if parent && child
+        child.with_relationship_type('genealogy') { child.update!(parent => parent) } if parent
```

Comment:
sorry - yes, use the old `if` that they had before

```ruby
child.parent = parent if parent && child
```

I was wrong - you are right.